work_queue/src/work_queue.c

```diff
@@ -697,6 +700,47 @@ work_queue_msg_code_t process_info(struct work_queue *q, struct work_queue_worke
 	return MSG_PROCESSED;
 }
 
+/*
+A cache-update message coming from the worker means that a requested
+remote transfer or command was successful, and know we know the size
+of the file for the purposes of cache storage management.
+*/
+
+int process_cache_update( struct work_queue *q, struct work_queue_worker *w, const char *line )
+{
+	char cachename[WORK_QUEUE_LINE_MAX];
+	long long size;
+	long long transfer_time;
+	
+	if(sscanf(line,"cache-update %s %lld %lld",cachename,&size,&transfer_time)==3) {
```

Comment:
update protocol number in work_queue_protocol.h

Reply:
Already went to version 11 in this PR.